utils/server_handler.py

```diff
@@ -0,0 +1,125 @@
+import http.client
+import subprocess
+import os
+
+DEFAULT_SERVER_LOCATION = "C:/Users/Matt/Dev/CS52-Autograder/serve-sml"
+DEFAULT_FILE_ORDER = ["buffer", "util", "http", "server", "assert", "grading_lib", "main"]
+
+class ServerHandler:
+
+    def __init__(self, server_location:str=None, files:list=None):
+        """Make the Server object
+
+        Args:
+        Server_location:  Where the sml files are located
+        files:  the files (and order) that they should be run in
+        """
+
+        if server_location is not None:
+            self.server_location = server_location
+        else:
+            self.server_location = DEFAULT_SERVER_LOCATION
+
+        if files is not None:
+            self.files = files
+        else:
+            self.files = DEFAULT_SERVER_LOCATION
+
+        self.process = None
+        self.has_started = False
+
+    def get_response(self, get_string:str) -> http.client.HTTPResponse:
```

Comment:
type hints get a space: `get_string: str`
but you probably want `Text` from the `typing` module instead of `str`